sw/device/tests/penetrationtests/firmware/ibex_fi.c

```diff
@@ -759,107 +945,42 @@ status_t handle_ibex_fi_char_unconditional_branch(ujson_t *uj) {
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
-  result = increment_counter(result);
+  // Attack target.
+  asm volatile("jal ra, increment_counter");
```

Comment:
can this be replaced by some code-gen macros: https://stackoverflow.com/a/28232095 ?